test_scripts/Resumption/InteriorVehicleData/023_InteriorVD_double_subscription_without_moduleId.lua

```diff
@@ -0,0 +1,49 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0188-get-interior-data-resumption.md
+--
+-- Description: SDL sends GetInteriorVehicleData response with WARNINGS result code in case of double subscription
+--  without moduleId
+--
+-- Precondition:
+-- 1. HMI and SDL are started
+-- 2. Mobile app with REMOTE_CONTROL hmi type is registered and activated
+-- 3. App is subscribed to the module_1
+--
+-- Sequence:
+-- 1. GetInteriorVehicleData(subscribe = true, module_1) is requested
+-- SDL does:
+-- - a. not send RC.GetInteriorVehicleData request to HMI
+-- - b. send GetInteriorVehicleData ("WARNINGS") response to mobile App
```

Comment:
resultCode is warning or info string is warning?
send GetInteriorVehicleData response (success=true, what is warning, isSubscribed=true ) response to mobile App

Reply:
@yang1070 It is resultCode = "WARNINGS"